db/dynamodb.py

```diff
@@ -183,6 +192,17 @@ def __create_table(self, table_name: str, key_type: str = 'S'):
             }
         )
 
+    def __enable_ttl(self, table_name: str):
+        """
+        Enable automatically dropping entries on TTL expiry
+
+        **Note**: This function should **not** be called externally, and should
+        only be called on initialization.
+
+        :param table_name: name of the table to enable ttl for
+        """
+        # Stub
```

Comment:
```suggestion
        # TODO
```

(for discoverability of TODOs by searching "TODO")

Should include links to the relevant issues next to the TODOs when you create them!